doc/api/tls.md

```diff
@@ -1641,8 +1641,8 @@ changes:
     More information can be found in the [RFC 4279][].
   * `ALPNProtocols`: {string\[]|Buffer\[]|TypedArray\[]|DataView\[]|Buffer|
     TypedArray|DataView}
-    An array of strings, `Buffer`s or `TypedArray`s or `DataView`s, or a
-    single `Buffer` or `TypedArray` or `DataView` containing the supported ALPN
+    An array of strings, `Buffer`s, `TypedArray`s, or `DataView`s, or a
+    single `Buffer`, `TypedArray`, or `DataView` containing the supported ALPN
```

Comment:
Doesn't have to be in this PR, but it's probably worth finding another way to express this, or maybe removing the list of possible types entirely as the types are described immediately above.